esx_service/utils/auth_api.py

```diff
@@ -93,10 +93,18 @@ def create_tenant_in_db(name, description, vms, privileges):
         error_info = error_code.generate_error_info(ErrorCode.TENANT_ALREADY_EXIST, name)
         return error_info, None
 
-    error_msg, tenant = auth_mgr.create_tenant(name=name,
-                                                description=description,
-                                                vms=vms,
-                                                privileges=privileges)
+    if name == auth.DEFAULT_TENANT:
```

Comment:
I'd like to repeat my previous comment - I do think we should handle the tenant uuid inside the create_tenant() API. IMO, uuid should not be handled by the API users. It's the API's responsibility to take care of the appropriate uuids.

If you recall, exposing uuid to the API users has caused multiple unnecessary changes.